studio/lib/queries/pages.ts

```diff
@@ -1,87 +1,105 @@
 import { groq } from "next-sanity";
 
-// TODO: set all sectons.ref's to slug.current
+export const LANGUAGE_FIELD_FRAGMENT = groq`
+  "language": $language
+`;
+
+export function translatedFieldFragment(fieldName: string) {
```

Comment:
same here. move to a util file instead of placing them under queries??

I see why you've placed it here, but because it's a function it doesn't really fit here.

Reply:
Will move this to a util file, but mostly because it will be reused in other query files. In general, I don't think we should split up related concepts just because they are functions or variables.